src/jti_content.js

```diff
@@ -1,169 +1,252 @@
+                    var StartPos = FindNextTI(cursorStart, tagStartIndex, tagEndIndex); // Find the starting <TI> tag
+                    $(this)[0].setSelectionRange(StartPos.start, StartPos.end);
+                    selectStart = StartPos.start; // Set Start Index
+                    selectEnd = StartPos.end; // Set End Index
+                } else { // Select next <TI> set
+                    if (tagStartIndex.indexOf(selectStart) === tagStartIndex.length - 1 && tagEndIndex.indexOf(selectEnd) === tagEndIndex.length - 1) { // Currently selecting the last set of <TI>, back to first set
+                        $(this)[0].setSelectionRange(tagStartIndex[0], tagEndIndex[0]);
+                        selectStart = tagStartIndex[0];
+                        selectEnd = tagEndIndex[0];
+                    } else {
+                        if (tagStartIndex.indexOf(selectStart) === -1 && tagEndIndex.indexOf(selectEnd) === -1) { // Highlighted <TI> tag is modified by user. Now we need search for the next <TI>.
+                            if (cursorStart < selectStart) cursorStart = selectStart;
+                            StartPos = FindNextTI(cursorStart, tagStartIndex, tagEndIndex); // Find the starting <TI> tag
+                            $(this)[0].setSelectionRange(StartPos.start, StartPos.end);
+                            selectStart = StartPos.start; // Set Start Index
+                            selectEnd = StartPos.end; // Set End Index
+                        } else {
+                            $(this)[0].setSelectionRange(tagStartIndex[tagStartIndex.indexOf(selectStart) + 1], tagEndIndex[tagEndIndex.indexOf(selectEnd) + 1]); // Find next set of <TI>
+                            selectStart = tagStartIndex[tagStartIndex.indexOf(selectStart) + 1];
+                            selectEnd = tagEndIndex[tagEndIndex.indexOf(selectEnd) + 1];
+                        }
+                    }
+                }
+            }
+        }
+    });
+});
+
+// Helper method. Return the keyCode of either ctrl or cmd based on OS
+function getKeyCodeOfCtrlOrCmd () {
+    var ctrlKey = 17,
+        cmdKey = 91;
+    if (navigator.appVersion.indexOf('Win') !== -1) return ctrlKey;
+    if (navigator.appVersion.indexOf('X11') !== -1) return ctrlKey;
+    if (navigator.appVersion.indexOf('Linux') !== -1) return ctrlKey;
+    if (navigator.appVersion.indexOf('Mac') !== -1) return cmdKey;
+}
+
+// Helper method. Find next <TI> based on cursor position
+function FindNextTI (CursorPos, tagStart, tagEnd) {
+    for (var i = 0; i < tagStart.length; i++) {
+        if (tagStart[i] >= CursorPos) {
+            return { start: tagStart[i], end: tagEnd[i] };
+        }
+    }
+    return { start: tagStart[0], end: tagEnd[0] };
+}
+
+// Helper method. Find index(start and end) of all occurrences of a given substring in a string
+function getAllIndexes (str) {
+    var startIndexes = [],
+        endIndexes = [];
+    var re = /<TI>/g, // Start

```

Comment:
You can probably just add the `re` and the `match` variables to the list of the first var declaration.